gateway/ln-gateway/tests/integration_tests.rs

```diff
@@ -602,7 +602,7 @@ async fn test_gateway_register_with_federation() -> anyhow::Result<()> {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn test_gateway_cannot_pay_expired_invoice() -> anyhow::Result<()> {
-    single_gateway_test(
+    single_federation_test(
```

Comment:
**thought:** "single" here is confusing. Is it single test, single federation... or single peer in a federation? :D

Reply:
For context, this test scenario involves the gateway and just one federation. The alternative test scenario where a gateway interacts with many federations is named `multi_federation_test`